translations/expansion_content.tex/de.po

```diff
@@ -69,20 +69,18 @@ msgstr ""
 
 #. type: multicols*
 #: sections/expansion_content.tex:10
-#, fuzzy, no-wrap
-#| msgid ""
-#| "\\begin{center}\n"
-#| "  \\includegraphics[width=0.55\\linewidth]{\\art/wyvern.png}\n"
-#| "\\end{center}\n"
-#| "\n"
+#, no-wrap
 msgid ""
 "\\begin{center}\n"
 "  \\includegraphics[width=\\linewidth,height=\\textheight,keepaspectratio]{\\art/efreet.jpg}\n"
 "\\end{center}\n"
 "\\columnbreak\n"
 "\n"
 msgstr ""
-"\\empty\n"
+"\\begin{center}\n"
```

Comment:
This doesn't hurt but not really necessary - if the translation is empty "", then po just uses the English version